documentation/source/faq/main.rst

```diff
@@ -0,0 +1,174 @@
+Frequently asked questions
```

Comment:
A good behavior of an FAQ is to add a list of all questions at the top and then link to the answers below.

Reply:
Done.